docs/av-stack-sync.1.md

```diff
@@ -0,0 +1,61 @@
+# av-stack-sync 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-sync - Synchronize stacked branches
+
+# SYNOPSIS
+
+`` av stack sync [--all | --current] [--no-push] [--no-fetch] [--prune]
+              [--trunk] [--continue | --abort | --skip] [--parent=<parent>]
+``
+
+# DESCRIPTION
+
+Synchronize stacked branches to be up-to-date with their parent branches.
+
+By default, this command will sync all branches starting at the root of the
+stack and recursively rebasing each branch based on the latest commit from the
+parent branch.
+
+If the --current flag is given, this command will not recursively sync dependent
```

Comment:
can we start explaining this with what this does vs what it does not do:

```
If the --current flag is given, this command will only sync stack up to the current branch starting from the first branch in the stack. It will not recursively sync ...
```

Reply:
Done.